libucontext.yaml

```diff
@@ -44,10 +45,15 @@ subpackages:
 
   - name: libucontext-doc
     pipeline:
+      - runs: |
+          make docs install_docs DESTDIR="${{targets.contextdir}}"
       - uses: split/manpages
     description: libucontext manpages
 
 update:
   enabled: true
-  release-monitor:
-    identifier: 17085
+  github:
```

Comment:
Could we also add a test here?

Reply:
a test to validate if `libucontext.so` is in the apk?